srcpkgs/shiromino/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'shiromino'
+pkgname=shiromino
+version=0.beta3.pre2
```

Comment:
this version seems to be beta

Reply:
the only tags available are beta unfortunately.  feel free to reject if it's not good enough for packaging